htdocs/web_portal/static_html/Help_And_Contact.html

```diff
@@ -6,49 +6,52 @@ <h1>Documentation, Help and Support</h1>
+        <a href="https://github.com/GOCDB/gocdb/blob/dev/INSTALL.md">Download, Install</a>
         </p>
              
     <h2>Need information?</h2> 
     <p>
-      Please browse  
-      <a href="https://wiki.egi.eu/wiki/GOCDB">
-      GOCDB Documentation Index</a> on the EGI wiki.
+      Please browse the GOCDB 
+      <a href="https://gocdb.github.io/">
+      Documentation </a> or your community documentation source.
       <br/><br/>
     </p>
     
     <h2>Found a bug?</h2> 
     <p> 
-      First, <a href='https://rt.egi.eu/rt/Dashboards/5541/GOCDB-Requirements' target='_blank'>
-      check known bugs and feature requests</a> to see if this has not 
-      already been reported.<br/> 
-      If not, please raise a ticket against GOCDB in
-      <a href='https://ggus.eu' target='_blank'> 
-      GGUS.</a>
+      First, check our 
+      <a href='https://github.com/GOCDB/gocdb/issues?q=is%3Aissue+is%3Aopen+label%3Abug' target='_blank'>
+      known bugs</a> to see if this has not already been reported.<br/> 
+      If not, please open a request in your community support desk or our 
+      <a href='https://github.com/GOCDB/gocdb/issues' target='_blank'> 
+      GitHub issues </a> 
+      page, if the former is not appropriate.
       <br/><br/> 
     </p> 
     
     <h2>Want to suggest something?</h2> 
     <p> 
       Before you make any request, check this is not already integrated to our development plans, see 
-      <a href='https://wiki.egi.eu/wiki/GOCDB/Roadmap' target='_blank'>GOCDB Development Plans.</a><br/> 
-      Any suggestion, new feature or improvement request should be submitted to the 
-      <a href='https://rt.egi.eu/rt/Dashboards/5541/GOCDB-Requirements' target='_blank'> 
-      RT requirements tracker</a>. <br/> 
-      Suggestions will be discussed within GOCDB developers, the OTAG, or any political 
-      body involved before inclusion into development plans.
+      <a href='https://github.com/GOCDB/gocdb/milestones' target='_blank'>GOCDB Development Plans.</a><br/> 
+      Any suggestion, new feature or improvement request should 
+      be submitted to your community support desk or our 
+      <a href='https://github.com/GOCDB/gocdb/issues' target='_blank'> 
+      GitHub issues </a>
+      page, if the former is not appropriate.<br/> 
+      Suggestions will be discussed by the GOCDB developers and 
+      any relevant project body before inclusion into development plans. 
       These bodies reserve the right to decline unsuitable requests.
       <br/><br/> 
     </p> 
     
     <h2>Need some support?</h2>
     
       <p>
-      For all other enquiries including general questions, temporary problems reports or 
-      support requests please  
-      <a href='https://ggus.eu' target='_blank'>submit a ticket in the GGUS system</a> 
+      For all other enquiries including general questions, temporary 
+      problems reports or support requests please open a 
```

Comment:
```suggestion
      For all other enquiries including general questions, temporary 
      problem reports or support requests please open a 
```

Maybe this would read better without the 's'?

Reply:
agreed